taas/mapping.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Mapping(object):
+    """
+        This class handles mappings between our raw input, and what we generate.
+    """
+
+    # We're an abstract class
+    __metaclass__ = ABCMeta
+
+    # This was originally called 'value', but apparently that's a
```

Comment:
It is a note-to-self. I'm doomed to repeat mistakes I don't document, and there's an excellent chance in a few months time I'll say "Why is this called emit? That's a silly name. We should call it value!".

We can remove it if you like, but leaves our class a cold, sterile place, devoid of personal struggles and triumphs. :)